packages/workflow/src/utils.ts

```diff
@@ -30,19 +30,22 @@ export const deepCopy = <T>(source: T): T => {
 	return clone;
 };
 // eslint-enable
-type ErrorMessage = { errorMessage?: string };
-type FallbackValue = { fallbackValue?: any };
-export const jsonParse = <T>(jsonString: string, options?: ErrorMessage | FallbackValue) => {
+
+interface JSONParseOptions<T> {
+	errorMessage?: string;
+	fallbackValue?: T;
+}
+
+export const jsonParse = <T>(jsonString: string, options: JSONParseOptions<T> = {}) => {
```

Comment:
reason for adding those types was to make the mutually exclusive, because as Ivan pointed it will not make sense to send message and fallback, your changes cancel that

Reply:
I'll revert that.